node_modules/README

```diff
@@ -1,9 +0,0 @@
-# Cockpit Javascript Dependencies
```

Comment:
Argh, this slipped in again.. Can you please FP again to restore the file?

Reply:
Oof. That awful file. Sorry. It feels like I'm constantly fighting it every single time I make any changes.